packages/react/src/components/TimePicker/TimePicker.tsx

```diff
@@ -0,0 +1,150 @@
+import * as React from 'react';
```

Comment:
The way this component is currently set up, it's not going to be targetable by theming/customization (ComboBox doesn't demonstrate the proper pattern for this). That might be okay for now since it entirely inherits styles from ComboBox, but might need to be revisited later.

Reply:
Resolved by tracking theming/customization for future reference.